include/hw/acpi/pc-hotplug.h

```diff
@@ -34,5 +34,6 @@
 
 #define PIIX_ACPI_PCI_HOTPLUG_IO_BASE 0xae00
 #define VIRT_ACPI_PCI_HOTPLUG_IO_BASE 0xae00
```

Comment:
@liujing2 these should be moved into the virt.h if possible. This will make the memory layout self explanatory.

Reply:
Fixed.